internal/pkg/gateway/server/test/utils.go

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Having a package to contain constructors for mocks is overkill. You're also building a package that I promise will eventually lead to import cycles.

Reply:
I've removed this package. Thanks :)